src/ioctls/vcpu.rs

```diff
@@ -2559,10 +2668,22 @@ mod tests {
                 .errno(),
             badf_errno
         );
+        #[cfg(any(
+            target_arch = "x86",
+            target_arch = "x86_64",
+            target_arch = "arm",
+            target_arch = "aarch64"
+        ))]
         assert_eq!(
             faulty_vcpu_fd.get_vcpu_events().unwrap_err().errno(),
             badf_errno
         );
+        #[cfg(any(
+            target_arch = "x86",
```

Comment:
No need for 32bit targets in these cfgs (actually, KVM on ARM32 hosts went away in 5.7 I think: https://www.phoronix.com/news/Linux-5.7-Kill-32-bit-ARM-KVM)

Reply:
Mh, I see we still have those all over the place... might also be worth a separate cleanup to just get rid of them all. I made a note for myself about this, feel free to ignore this comment for now, as its just a "fitting in with existing code" kinda thing.

Reply:
Yes, I was about to raise a PR to drop unsupported architectures (or not tested) not just limited to `x86` (`s390` and etc. as well) :)

Reply:
I think the s390x and ppc ones are fine, because there's folks that use them and also provide fixes if we break stuff, so it's sort of "unofficially maintained", to some capacity. The 32bit ones can definitely go though imo (ARM32 because it doesnt exist anymore, and x86 because we're already very explicit about not supporting 32 bit in other rust-vmm crates)